setup.py

```diff
@@ -50,7 +50,7 @@ def get_readme():
 VERSION, DEV_CLASSIFIER = get_version_info()
 
 setup(
-    name='iodata',
+    name='qc-iodata',
```

Comment:
Do we need to change the name here? I´m asking because the package name would be maintained as iodata, right?

Reply:
Yes, the only serious role of this string is to determine the name of the package on https://pypi.org/. It also affects the filenames of distribution files generated by `setup.py`. The package name for importing remains the same. `import iodata` still works as usual.